docs/data-sources/infoblox_a_record.md

```diff
@@ -11,7 +11,7 @@ To get information about an A-record, specify a combination of the DNS view, IPv
 
 The following list describes the parameters you must define in an `infoblox_a_record` data source block (all of them are required):
 
-* `dns_view`: the DNS view in which the zone exists.
+* `dns_view`: optional, specifies the DNS view in which the reverse mapping zone exists. If a value is not specified, the name `default` is used as the DNS view.
```

Comment:
Could we update the first sentence as 'Specifies the DNS view in which the record's zone exists.'?